From 1ff1a6df197dfe4b70c5ed446b5349448376139a Mon Sep 17 00:00:00 2001
From: Rajesh Bhagat <rajesh.bhagat@nxp.com>
Date: Thu, 1 Feb 2018 03:34:47 +0530
Subject: [PATCH 10/31] board: ls1043adcm: Move CONFIG_ENV_IS_IN_SPI_FLASH for
 platform header files to defconfig

save command was not available in ls1043adcm u-boot, this patch fixed the same

Signed-off-by: Rajesh Bhagat <rajesh.bhagat@nxp.com>
---
 configs/ls1043adcm_defconfig | 1 +
 include/configs/ls1043adcm.h | 1 -
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/ls1043adcm_defconfig b/configs/ls1043adcm_defconfig
index 2bf109643d..cc93553315 100644
--- a/configs/ls1043adcm_defconfig
+++ b/configs/ls1043adcm_defconfig
@@ -18,6 +18,7 @@ CONFIG_CMD_USB=y
 CONFIG_CMD_CACHE=y
 CONFIG_MTDPARTS_DEFAULT="mtdparts=60000000.nor:2m@0x100000(nor_bank0_uboot),40m@0x1100000(nor_bank0_fit),7m(nor_bank0_user),2m@0x4100000(nor_bank4_uboot),40m@0x5100000(nor_bank4_fit),-(nor_bank4_user);7e800000.flash:1m(nand_uboot),1m(nand_uboot_env),20m(nand_fit);spi0.0:1m(uboot),5m(kernel),1m(dtb),9m(file_system)"
 CONFIG_OF_CONTROL=y
+CONFIG_ENV_IS_IN_SPI_FLASH=y
 CONFIG_DM=y
 CONFIG_FSL_CAAM=y
 CONFIG_SPI_FLASH=y
diff --git a/include/configs/ls1043adcm.h b/include/configs/ls1043adcm.h
index d426bb6c28..e70a18bc41 100644
--- a/include/configs/ls1043adcm.h
+++ b/include/configs/ls1043adcm.h
@@ -254,7 +254,6 @@
 #define CONFIG_SYS_MMC_ENV_DEV		0
 #define CONFIG_ENV_SIZE			0x2000
 #elif defined(CONFIG_QSPI_BOOT)
-#define CONFIG_ENV_IS_IN_SPI_FLASH
 #define CONFIG_ENV_SIZE			0x2000		/* 8KB */
 #define CONFIG_ENV_OFFSET		0x300000	/* 3MB */
 #define CONFIG_ENV_SECT_SIZE		0x40000		/* 256KB */
-- 
2.17.1

